app/assets/javascripts/webchat/library.js

```diff
@@ -27,14 +23,19 @@
       if (!availabilityUrl || !openUrl) {
         throw Error.new('urls for webchat not defined')
```

Comment:
Not your code, but I wonder if we should take this opportunity to improve this error handling. Kevin suggested something like this on a bit of work I did recently:

```
throw Error.new('urls for webchat not defined', window.location)
```

Reply:
Would passing `window.location.href` be more useful than using `window.location`? The `window.location` object seems to have a lot of stuff in it:

![Screenshot 2022-03-08 at 12 00 59](https://user-images.githubusercontent.com/5793815/157234526-092b9c87-bf26-4686-a878-59629e95cd8f.png)

Reply:
Either's good.

Reply:
I'll add this change to a new commit.